wms/src/main/java/bg/tuvarna/sit/MainApp.java

```diff
@@ -14,10 +19,17 @@ public class MainApp extends Application {
   @Override
   public void start(Stage stage) throws IOException {
 
-    FXMLLoader fxmlLoader = new FXMLLoader(MainApp.class.getResource("/views/scene.fxml"));
-    Scene scene = new Scene(fxmlLoader.load(), 320, 240);
-    stage.setTitle("Hello!");
-    stage.setScene(scene);
+//    FXMLLoader fxmlLoader = new FXMLLoader(MainApp.class.getResource("/views/scene.fxml"));
+//    Scene scene = new Scene(fxmlLoader.load(), 320, 240);
+//    stage.setTitle("Hello!");
+//    stage.setScene(scene);
+//    stage.show();
+
+    FXMLLoader loader = new FXMLLoader(getClass().getResource("/views/warehouseControlPanel.fxml"));
```

Comment:
Here we should load the Home page because it is the entry point of the app.
We can add a button instead that will link to the warehouse control panel after the user is logged in and has the required permissions (role) 
https://github.com/TU-Varna-SIT/warehouse-management-system/blob/master/wms/src/main/java/bg/tuvarna/sit/MainApp.java#L37